cloudify_cli/commands/agents.py

```diff
@@ -109,16 +119,20 @@ def create_deployments_list(client, deployment_id, logger):
     :return A list of the relevant deployments.
             """
     # install agents for a specified deployment
+    raise_error = False
```

Comment:
This flag is actually not really necessary, as you can decide on whether there was an error or not, just by looking at the error message. If `log_msg` is None, you know there was no error, and if it's anything other than None, you there was. So you can only return 2 values - deps and log_msg